package.json

```diff
@@ -40,6 +47,7 @@
   },
   "devDependencies": {
     "almond": "^0.3.1",
+    "builder": "FormidableLabs/builder#feature-setup-cleanup2",
```

Comment:
TODO: Need new `builder` release once https://github.com/FormidableLabs/builder/pull/55 is merged.